benchmark/realnest/03_create_table_from_unnested_structs.benchmark

```diff
@@ -0,0 +1,43 @@
+# name: benchmark/realnest/03_create_table_from_unnested_structs.benchmark
+# description: Create a table combining all the fields from a JSON file, unnesting all structures
```

Comment:
`Create a table by unnesting and joining the structs in a JSON file`

Reply:
resolved